bears/vcs/CommitBear.py

```diff
@@ -150,6 +157,11 @@ def run(self,
                 yield Result(self, 'HEAD commit has no message.')
             return
 
+        result_message = self.check_skip_ci_build(
```

Comment:
this type of design is exactly what we dont want.  this bear has way too much logic in it already.